demo/index.js

```diff
@@ -293,6 +294,23 @@ app.get('/settlements', function (req, res) {
   });
 });
 
+app.get('/exchange-rate', (req, res) => {
+  const {
+    fee_type, //string 'USD'
+    date, //string '20180801'
+    // visit https://pay.weixin.qq.com/wiki/doc/api/external/jsapi.php?chapter=9_15&index=10 for more info
+    } = req.query;
+  wx.payment.queryexchagerate({
+    fee_type: fee_type || 'GBP',
+    date: date || utils.simpleDate(new Date(), 'yyyyMMdd'),
```

Comment:
the format should be `yyyymmdd` in [dateformat package](https://github.com/felixge/node-dateformat/tree/v2_2), the capital `MM` is for minutes